data/websites.json

```diff
@@ -540,5 +550,6 @@
       "url": "https://daorupandehi.moha.gov.np/en",
       "description": "Official Website Of District  Administration Office Of Rupandehi"
     }
+    
   ]
-}
+}
```

Comment:
There's still no new line at the end of the file.
Please format the files using prettier and push the commit again.